docs/spec/index.md

```diff
@@ -1,54 +1,67 @@
-| to_trip_id | ID referencing [**trips.trip_id**](https://developers.google.com/transit/gtfs/reference#tripstxt) | Conditionally Required | Identifies the trip scheduled immediately following to the deadhead within the block_id. |
-| from_trip_id | ID referencing [**trips.trip_id**](https://developers.google.com/transit/gtfs/reference#tripstxt) | Conditionally Required | Identifies the trip scheduled immediately prior to the deadhead within the block_id. |
-| to_deadhead_id | ID referencing **deadheads.deadhead_id** | Conditionally Required | Identifies the deadhead scheduled immediately following the deadhead within the block_id. |
-| from_deadhead_id | ID referencing **deadheads.deadhead_id** | Conditionally Required | Identifies the deadhead scheduled immediately prior to the deadhead within the block_id. |
 
-### ops_locations.txt
+### Files
 
-| **Field Name** | **Type** | **Required** | **Description** |
-| --- | --- | --- | --- |
-| ops_location_id | ID | Required | Identifies an operational location. |
-| ops_location_code | String | Optional | Short text or a number that identifies the operational location for internal use. |
-| ops_location_name | String | Required | Name of the operational location. |
-| ops_location_desc | String | Optional | Description of the operational location. |
-| ops_location_lat | Latitude | Required | The latitude of the operational location. |
-| ops_location_lon | Longitude | Required | The longitude of the operational location. |
+| **File Name** | **Type** | **Description** |
+| --- | --- | --- |
+| trips_supplement.txt | Supplement | Supplements and modifies GTFS [trips.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#tripstxt) with deadheads and other non-public trip information.|
+| stops_supplement.txt | Supplement | Supplements and modifies GTFS [stops.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#stopstxt) with internal stop locations, waypoints, and other non-public stop information.|
+| stop_times_supplement.txt | Supplement | Supplements and modifies GTFS [stop_times.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#stop_timestxt) with non-public times that trips stop at locations, and related information. |
+| routes.txt | Supplement | Supplements and modifies GTFS [routes.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#routestxt) with internal route identifiers and other non-public route identification. |
+| runs_pieces.txt | TODS-Specific | Defines daily personnel schedules within a feed. |
+| run_events.txt | TODS-Specific| Defines other scheduled activities to be performed by a member of personnel during a run. |
 
-### deadhead_times.txt
+The use of the `_supplement` standard to modify other GTFS files remains experimental at this time.
+
+
+## `_supplement` Files
```

Comment:
I think we should keep the practice of having individual tables for each file, even if the `_supplement` files have the same information for now.

Reply:
As long as the tables don't have to list every GTFS column, and can elide them. Listing out all the columns in all the files and fields would be so much text that it'd make it hard to see where the new and interesting parts of TODS are, and would make it harder to understand the shared ideas behind the supplement files.

Example:

`trips_supplement.txt`

<table>
<tr>
<th>Field name</th>
<th>Type</th>
<th>Required</th>
<th>Description</th>
</tr>
<tr>
<td>... any column in GTFS `trips.txt`</td>
<td>same values as in GTFS</td>
<td>Conditionally Required</td>
<td>
Required if adding new data to GTFS, and the GTFS column is required.
Optional if the GTFS field is optional, or if it's modifying a required row that already exists GTFS.
Should be blank if `TODS_delete` is `1`.
</td>
</tr>
<tr>
<td><code>TODS_trip_type</code></td>
<td>Text</td>
<td>Optional</td>
<td>
Defines the type of the trip if distinct from a standard revenue trip.
</td>
</tr>
</tr>
<tr>
<td><code>TODS_delete</code></td>
<td>Enum</td>
<td>Optional</td>
<td>
If <code>1</code>, removes the row from GTFS instead of modifying it.
</td>
</tr>
</table>

Although I think that even having 4 near-identical tables for the supplement files might make it hard to spot the differences.

Reply:
My thought process behind this was to think from the perspective of someone directly interacting with the data.

By presenting it as, "all files of this type do the same thing from x to y," it guides a reader to implementing a common programatic framework for the parsing and processing of each file rather than a bespoke process specific to each file.

This was also the motivation behind the note about the experimental application to other files: basically hinting at the notion that subsequent file additions of this type will follow the same approach/methodology, so implementing a generalized approach will save effort in the long-term.

My concerns with listing each individual file are:
1. That's a lot of data to either replicate from GTFS or include that's redundant.
2. Similarities between files would be more difficult to spot (per Sky).
3. It could prompt developers to implement different processes for different files, which could present challenges in the future.
4. As additional files are added, redundancies would grow and grow, presenting additional opportunities for error.

Reply:
@jeffkessler-keolis @skyqrose 
**RE: TODS_delete:** In what cases is it necessary to remove (rather than supplement) passenger-facing information from the GTFS? This caught my eye out of a desire to keep implementation simple.

Reply:
We never identified such a use case; see https://github.com/cal-itp/operational-data-standard/pull/61#discussion_r1513587956 for the thread of that discussion.